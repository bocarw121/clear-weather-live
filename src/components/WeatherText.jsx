import React from "react";
import { StyleSheet, Text } from "react-native";

const WeatherText = ({ text, fontSize, color }) => {
  return <Text style={[styles.text, { fontSize, color }]}>{text}</Text>;
};

export default WeatherText;

const styles = StyleSheet.create({
  text: {
    marginTop: 10,
    textAlign: "center",
  },
});
