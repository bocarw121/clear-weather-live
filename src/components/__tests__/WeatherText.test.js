import React from "react";

import { render } from "@testing-library/react-native";

import WeatherText from "../WeatherText";

it("Should render the WeatherText", () => {
  const screen = render(<WeatherText />);

  expect(screen).toMatchSnapshot();
});
