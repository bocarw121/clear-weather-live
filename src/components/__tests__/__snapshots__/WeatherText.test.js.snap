// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the WeatherText 1`] = `
<Text
  style={
    Array [
      Object {
        "marginTop": 10,
        "textAlign": "center",
      },
      Object {
        "color": undefined,
        "fontSize": undefined,
      },
    ]
  }
/>
`;
